client/me/purchases/cancel-privacy-protection/index.jsx

```diff
@@ -84,20 +84,26 @@ const CancelPrivacyProtection = React.createClass( {
 
 		this.setState( {
 			disabled: true,
-			cancelling: true
+			cancelling: true,
 		} );
 
-		this.props.cancelPrivacyProtection( id ).then( () => {
-			this.resetState();
-
-			notices.success( this.translate( 'You have successfully canceled privacy protection for %(domain)s.', {
-				args: { domain }
-			} ), { persistent: true } );
-
-			page( paths.managePurchase( this.props.selectedSite.slug, id ) );
-		} ).catch( () => {
-			this.resetState();
-		} );
+		this.props
+			.cancelPrivacyProtection( id )
```

Comment:
I'm not sure I like this kind of formatting. I would rather have these two lines grouped:

```javascript
this.props.cancelPrivacyProtection( id )
```